src/pages/programs/programsView.js

```diff
@@ -1,21 +1,48 @@
+/* eslint-disable no-param-reassign */
 import React from 'react';
 import {
   Grid,
   withStyles,
 } from '@material-ui/core';
 import { CustomDataTable } from 'bento-components';
-import { Link } from 'react-router-dom';
 
 import TableFooter from '@material-ui/core/TableFooter';
 import TableRow from '@material-ui/core/TableRow';
 import TablePagination from '@material-ui/core/TablePagination';
 import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  table, tableTitle, icon, externalLinkIcon,
+} from '../../bento/programData';
 import Stats from '../../components/Stats/AllStatsController';
 import { Typography } from '../../components/Wrappers/Wrappers';
-import icon from '../../assets/trial/Trials_Title_Bar.Icon.svg';
-import externalLinkIcon from '../../assets/icons/Program-ExternalLink.svg';
 import { singleCheckBox, fetchDataForDashboardDataTable } from '../dashboard/dashboardState';
 
+function manipultateLinks(tableData) {
+  let cloumnCount = 0;
+  tableData.forEach((column) => {
```

Comment:
replace this with ableData.forEach((column, index) => {
index will go ve the columCount

Reply:
done